ipa-core/Cargo.toml

```
@@ -86,6 +86,8 @@ ipa-prf = []
 relaxed-dp = []
 
 [dependencies]
+ipa-metrics = { path = "../ipa-metrics", features = [] }
```

```suggestion
ipa-metrics = { path = "../ipa-metrics" }
```
Unless this is somehow needed to remove the partitions feature.